### PR TITLE
add codeowners

### PR DESCRIPTION
**Pull request by orouz**
fixes:
- [x] We need to add our team to the .github/CODEOWNERS file

from:
- https://github.com/elastic/security-team/issues/2580

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -435,6 +435,9 @@ x-pack/test/security_solution_cypress @elastic/security-engineering-productivity
 # Security Asset Management
 /x-pack/plugins/osquery @elastic/security-asset-management
 
+# Cloud Posture Security
+/x-pack/plugins/cloud_security_posture/ @elastic/cloud-posture-security
```
Comment (orouz) [inline .github/CODEOWNERS:439]: `cloud_security_posture` vs `cloud-posture-security`

we should pick one. probably changing the folder is easier than the team name. 

thoughts? 

@ari-aviran @kfirpeled

Comment (ari-aviran) [reply .github/CODEOWNERS:439]: No need for any changes, Kibana uses `snake_case` for directories and team names use a dash, this is fine and all other plugins act the same

**Review by ari-aviran — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (orouz) [reply .github/CODEOWNERS:439]: i meant that we use `posture` first for github team and `security` first for folder

Comment (ari-aviran) [reply .github/CODEOWNERS:439]: Oh... yea, GitHub team name feels wrong, I think we can live with this inconsistency

**Pull request merged**